src/js/controllers/chatController.js

```diff
@@ -69,45 +63,39 @@ module.exports = ViewController.extend({
 
     sendMessage: function(text) {
         var self = this;
-        return new Promise(function(resolve, reject) {
-            if (self.conversation.isNew()) {
-                self.conversation = self.collection.create(self.conversation, {
-                    success: resolve,
-                    error: reject
-                });
-            } else {
-                resolve(self.conversation);
-            }
-        })
+        return this._getConversation()
             .then(this._initFaye)
```

Comment:
I think you'll need to init faye after posting the message now, since there is no conversation id until the first message is posted, or the first whisper is received